assets/xml/scenes/dungeons/jyasinzou.xml

```diff
@@ -4,6 +4,8 @@
         <Texture Name="gSpiritTempleDayEntranceTex" OutName="day_entrance" Format="ia16" Width="8" Height="128" Offset="0x18920"/>
         <Texture Name="gSpiritTempleNightEntranceTex" OutName="night_entrance" Format="ia16" Width="8" Height="128" Offset="0x18020"/>
         <Scene Name="jyasinzou_scene" Offset="0x0"/>
+        <Scene Name="jyasinzou_scene_unused1" Offset="0x17200"/>
+        <Scene Name="jyasinzou_scene_unused2" Offset="0x176F0"/>
```

Comment:
does defining multiple Scenes in the same xml like this still include it in the same file? I dont think this system was designed to be used with this use case in mind, but if it works, it works

Reply:
yeah, it seems to work the same as declaring another texture or something

Reply:
works for me